### DISCUSSION TITLE
MPU6050 sensor for nrf5340pdk board?

### DISCUSSION THREAD

**Discussion by developerzephyros** [zephyrproject-rtos/zephyr#31221] [Q&A]
Hi,
I am using a MPU6050 sensor for my nrf5340pdk board from nordic semiconductors.
However, I face some problems with using this sensor for my board and running the given demo example.

In the example, the nrf52dk_nrf52832.overlay file is given for the nrf52dk_nrf52832 board so I simply renamed the file to nrf5340dk_nrf5340_cpuapp.overlay to use it for the nrf5340pdk board. Is it that easy, or am I missing something?
If I try to build that example, I always get the following error: 
![error_MPU6050_sensor](https://user-images.githubusercontent.com/77276329/104180411-5f976f00-540d-11eb-9cc5-6536624bedd3.PNG)

Does someone know, how I can solve that problem in order to use that sensor? Or is that simply not possible?
Thank you!!

**Comment by AnanthJaisankar** [comment]
Did you create necessary overlay file and linked it?
